CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
```

Comment:
@ota42y nitpick - shouldn't this be under the "released" section, since `1.0.0` is now released to `master`?